terraform/modules/env-health-checks/data_browser_and_portal.tf

```diff
@@ -0,0 +1,7 @@
+data "external" "data_browser_and_portal_health_check_id" {
```

Comment:
Almost there. You need to add this as an output in `outputs.tf` and put that health check id in the the aggregate alert https://github.com/HumanCellAtlas/dcp-monitoring/blob/master/terraform/modules/env-health-checks/dcp.tf#L6